generate-release-notes.sh

```diff
@@ -74,7 +74,7 @@ parse_args() {
 # Validates the options specified for the script.
 validate_args() {
     if [ -z "$AL2_GPU_CUDA_VERSION" ] || [ -z "$AL2_GPU_NVIDIA_VERSION" ]; then
-        if ! is_ami_excluded "al2gpu"; then
+        if ! { is_ami_excluded "al2gpu" && is_ami_excluded "al2kernel5dot10gpu"; }; then
```

Comment:
Q. Is it possible that, we only have `al2kernel5dot10gpu` in the release to patch kernel 5.10, and exclude al2gpu in that release? If this could happen, should it modify this validation to handle this case? Thanks!